### PR TITLE
(#5293) - fix SQLite Plugin usage, add test

### PR DESCRIPTION
**Pull request by nolanlawson**
We've regressed on this so many times, I've also added a test that mocks `window.sqlitePlugin` and ensures we are passing in the correct args.

### CODE REVIEW

**Comment by daleharvey** [comment]
Looks like this breaks the sql related tests

**Comment by nolanlawson** [comment]
Yup, my bad, forgot about the cache-by-name. Should be fixed now.

**Comment by daleharvey** [comment]
👍 looks good, cheers

File: tests/integration/browser.sqlite.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+// mock tests for the Cordova SQLite Plugin
+
+var adapters = ['local'];
```
Comment (daleharvey) [inline tests/integration/browser.sqlite.js]: If its only designed for one adapter, it doesnt need to do the adapters.forEach dance

**Comment by daleharvey** [comment]
I made that minor change on the way in, thanks - https://github.com/pouchdb/pouchdb/commit/b600bdd249946aa9eeabefbe868d9987c9240723

**Pull request closed without merging**